libcxx/include/__format/formatter_floating_point.h

```diff
@@ -690,7 +690,7 @@ __format_floating_point(_Tp __value, _FormatContext& __ctx, __format_spec::__par
       // Let P equal the precision if nonzero, 6 if the precision is not
       // specified, or 1 if the precision is 0. Then, if a conversion with
       // style E would have an exponent of X:
-      int __p = std::max(1, (__specs.__has_precision() ? __specs.__precision_ : 6));
+      int __p = std::max(int32_t(1), (__specs.__has_precision() ? __specs.__precision_ : 6));
```

Comment:
This should work too and based in the type of `__p` this looks more natural.
```suggestion
      int __p = std::max<int>(1, (__specs.__has_precision() ? __specs.__precision_ : 6));
```
Can you verify this works for you?

Reply:
> Can you verify this works for you?

It does, thanks!  Updated per suggestion.